tests/bluetooth/bsim_bt/bsim_test_audio/test_scripts/has.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2022 Codecoup
+#
+# SPDX-License-Identifier: Apache-2.0
+
+SIMULATION_ID="has"
+VERBOSITY_LEVEL=2
+PROCESS_IDS=""; EXIT_CODE=0
+
+function Execute(){
+  if [ ! -f $1 ]; then
+    echo -e "  \e[91m`pwd`/`basename $1` cannot be found (did you forget to\
+ compile it?)\e[39m"
+    exit 1
+  fi
+  timeout 20 $@ & PROCESS_IDS="$PROCESS_IDS $!"
+}
+
+: "${BSIM_OUT_PATH:?BSIM_OUT_PATH must be defined}"
+
+#Give a default value to BOARD if it does not have one yet:
+BOARD="${BOARD:-nrf52_bsim}"
+
+cd ${BSIM_OUT_PATH}/bin
+
+printf "\n\n======== Running HAS main (API) test =========\n\n"
+
+Execute ./bs_${BOARD}_tests_bluetooth_bsim_bt_bsim_test_audio_prj_conf \
+  -v=${VERBOSITY_LEVEL} -s=${SIMULATION_ID} -d=0 -testid=has -rs=24
+
+Execute ./bs_${BOARD}_tests_bluetooth_bsim_bt_bsim_test_audio_prj_conf \
+  -v=${VERBOSITY_LEVEL} -s=${SIMULATION_ID} -d=1 -testid=has_client -rs=46
+
+Execute ./bs_2G4_phy_v1 -v=${VERBOSITY_LEVEL} -s=${SIMULATION_ID} \
+  -D=2 -sim_length=20e6 $@
```

Comment:
Increase to >30e6, so that the test has a chance to time out.  
The other tests here are all 60e6 (except tbs, which was upmerged after this fix.)

See commits f7cd6afb79c6d013726909db15b988f98930f5b8 and 73f5ffcf4e207381df0bd40202839b8aa4c6a861